docs/articles/samples/IntroCustomEnvironmentInfo.md

```diff
@@ -0,0 +1,18 @@
+---
+uid: BenchmarkDotNet.Samples.IntroCustomEnvironmentInfo
+---
+
+## Sample: IntroCustomEnvironmentInfo
+
+You can add any useful information about environment to the benchmark report. Just mark one or several static methods that return `string` or `IEnumerable<string>` by the `[CustomEnvironmentInfo]` attribute.
```

Comment:
`[CustomEnvironmentInfo]` should be a hyperlink to the API reference.
An example is here: https://raw.githubusercontent.com/dotnet/BenchmarkDotNet/v0.11.1/docs/articles/samples/IntroParams.md